apps/studio/src/server/modules/user/inactiveUsers.service.ts

```diff
@@ -0,0 +1,198 @@
+import { ISOMER_ADMINS_AND_MIGRATORS_EMAILS } from "~prisma/constants"
+
+import type { User } from "../database"
+import type { AccountDeactivationEmailTemplateData } from "~/features/mail/templates/types"
+import { sendAccountDeactivationEmail } from "~/features/mail/service"
+import { createBaseLogger } from "~/lib/logger"
+import { db, RoleType, sql } from "../database"
+import { PG_ERROR_CODES } from "../database/constants"
+import { MAX_DAYS_FROM_LAST_LOGIN } from "./constants"
+
+export const HOURS_IN_MS = 60 * 60 * 1000
```

Comment:
[super nit] if exporting this, we should probably put in a shared `constants.ts` but this is v minor